pkg/apis/admissionregistration/validation/validation.go

```diff
@@ -183,6 +183,24 @@ func validateExternalAdmissionHook(hook *admissionregistration.ExternalAdmission
 	if hook.FailurePolicy != nil && *hook.FailurePolicy != admissionregistration.Ignore {
 		allErrors = append(allErrors, field.NotSupported(fldPath.Child("failurePolicy"), *hook.FailurePolicy, []string{string(admissionregistration.Ignore)}))
 	}
+
+	if len(hook.ClientConfig.URLPath) != 0 {
+		if !strings.HasPrefix(hook.ClientConfig.URLPath, "/") || !strings.HasSuffix(hook.ClientConfig.URLPath, "/") {
```

Comment:
requiring or forbidding trailing `/` is tricky, depending on the server implementing the webhook. why not allow either?

Reply:
> requiring or forbidding trailing / is tricky, depending on the server implementing the webhook. why not allow either?

done